app/classifier/tasks/survey/choice.cjsx

```diff
@@ -52,7 +52,7 @@ module.exports = React.createClass
     onCancel: Function.prototype
 
   getInitialState: ->
-    answers: {}
+    answers: @props.annotation?.answers || {}
```

Comment:
Is there a `propType` (and `defaultProp` if necessary) defined for `annotation`?